tests_e2e/scenario_utils/azure_models.py

```diff
@@ -0,0 +1,239 @@
+import time
```

Comment:
The files under 'scenario_utils' come straight from the 'dcr' POC, which in turn copied them from the current DCR implementation. They need significant improvements on error handling and diagnostics logging, feel free to skip them for this review.